canonical-web-and-design/ubuntu.com
# Add engage pages new language fallback

## Done

Currently, when engage pages are created with a new langauge, e.g. Russian (ru) it breaks the template.
This PR adds a fallback, so that if there is no template, it will fallback to the English version.
Still, we should translate and create the language-specific template once we see engage pages added with new languages.
This fallback also works for random language code, which may or may not be added mistakengly by the author of the engage page.

## QA

- Check https://ubuntu-com-11052.demos.haus/engage and click on the fictional engage page in Ukrainian language.
- Make sure you can see it and it works.
- Go back to https://ubuntu-com-11052.demos.haus/engage and click on the vimeo engage page and check that it works
- Go back to https://ubuntu-com-11052.demos.haus/engage and click on an engage page in a different language that has a form, and fill it up and test that it works. 
- If you have time, try a couple more in other languages, to ensure this new functionality doesn't break old functionality

Here is the source engage page created for testing https://discourse.ubuntu.com/t/test-unsupported-language-engage-page/25802
And here is the test index topic https://discourse.ubuntu.com/t/test-engage-pages-and-takeovers/25747 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11037

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
